pkgs/tools/security/hashcat-utils/default.nix

```diff
@@ -0,0 +1,28 @@
+{ stdenv, fetchFromGitHub }:
+
+stdenv.mkDerivation rec {
+  name = "hashcat-utils-${version}";
+  version = "1.9";
+
+  src = fetchFromGitHub {
+    owner = "hashcat";
+    repo = "hashcat-utils";
+    rev = "v${version}";
+    sha256 = "0wgc6wv7i6cs95rgzzx3zqm14xxbjyajvcqylz8w97d8kk4x4wjr";
+  };
+
+  sourceRoot = "source/src";
+
+  installPhase = ''
```

Comment:
Instead you can do
```nix
  installPhase = ''
    runHook preInstall
    install -Dm0555 *.bin -t $out/bin
    install -Dm0555 *.pl -t $out/bin
    runHook postInstall
  '';
```

This also install the perl script tools.
Also: do we want to keep the extension `.bin` and `.pl` for each command?

Reply:
> This also install the perl script tools.

I did not even notice them :see_no_evil: Thanks for pointing that (and the other things) out!

> Also: do we want to keep the extension .bin and .pl for each command?

Some people apparently use them with extension, others without. I am inclined to keep the extensions as the hashcat wiki ( https://hashcat.net/wiki/doku.php?id=hashcat_utils#cap2hccapx ) shows them that way.